src/plugins/blocksmith/utils/isAdvancedBlockSelected.js

```diff
@@ -12,20 +12,5 @@ import getSelectedBlocksList from './getSelectedBlocksList';
  */
 export default (editorState) => {
```

Comment:
@aloncohenn since this is simpler now i would rename the util to isAtomicBlockSelected

Reply:
@efjacobson renamed